worker/exec/konnector_test.go

```diff
@@ -410,6 +431,20 @@ echo "{\"type\": \"toto\", \"message\": \"COZY_URL=${COZY_URL}\"}"
 		var updatedAcc account.Account
 		err = couchdb.GetDoc(inst, consts.Accounts, acc.ID(), &updatedAcc)
 		require.NoError(t, err)
-		assert.Equal(t, updatedAcc.DefaultFolderPath, "/Administrative/my-konnector-1/account-1")
+		assert.Equal(t, updatedAcc.DefaultFolderPath, "/Administrative/Trainline/account-1")
 	})
 }
+
+func loadLocale() error {
```

Comment:
You can take `t *testing.T` as an argument and mark the function as an helper with `t.Helper()`